main/src/ca/uwaterloo/flix/api/lsp/Position.scala

```diff
@@ -68,6 +70,8 @@ case class Position(line: Int, character: Int) extends Ordered[Position] {
   // NB: LSP line and column numbers are zero-indexed, but Flix uses one-indexed numbers internally.
   def toJSON: JValue = ("line" -> (line - 1)) ~ ("character" -> (character - 1))
 
+  def toLsp4j: lsp4j.Position = new lsp4j.Position(line - 1, character - 1)
```

Comment:
I would copy this comment from above:

// NB: LSP line and column numbers are zero-indexed, but Flix uses one-indexed numbers internally.